schemas/event.yml

```diff
@@ -99,45 +220,108 @@
       example: user-password-change
 
     - name: outcome
-      level: extended
+      level: core
       type: keyword
-      short: The outcome of the event.
+      short: The outcome of the event. (Categorization Field)
       description: >
-        The outcome of the event.
+        This is one of four ECS Categorization Fields, and indicates the
```

Comment:
Not that big of a deal, but wondering if this fully makes sense to everyone contextually? "This is one of..." in the context of "Allowed Values"? Wondering if we want to change the heading docs in a subsequent PR, maybe make the page header be something like `ECS Categorization Field: event.outcome` and then have "Table of Contents" say something like "Allowed Values"

Just raising this here since the "description" of the field makes sense to me, but it doesn't quite seem to fit the context of where it is on the particular page, since I'd assume at that point we'd already be talking about the "Allowed Values" rather than the field itself.

<img width="735" alt="Screen Shot 2019-12-16 at 11 14 16 AM" src="https://user-images.githubusercontent.com/3577250/70923066-40a19000-1ff5-11ea-8134-16c64a438cd0.png">

Reply:
The heading of that documentation section is indeed hidden far away from these 4 pages (it's in the breadcrumbs and the sidebar only).

I'll experiment with bringing this front and center on each of the 4 pages. Good point 👍

Reply:
I think it looks a little funny in the sidebar, but it works better on the page itself:

<img width="1170" alt="ECS Categorization Field in each page heading" src="https://user-images.githubusercontent.com/2300/70945441-35655900-2023-11ea-8c23-3f2bc47b938b.png">

Reply:
That's fine, still think we should circle back at some point and figure out something that works a bit better contextually--but yeah, the navigation looks nasty with that convention.

Reply:
I just realized I hadn't renamed the heading before the list... Adjusting now.